### PR TITLE
Update delivery report query

### PR DESCRIPTION
**Pull request by ivadym**
## Description

Since the list of analysis to generate delivery reports is now pipeline specific, some minor changes are required to automate it.

### Changed
- Delivery report query: retrieving analysis only for a specific pipeline

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by ivadym** [comment]
## Tests on stage:

### Previous query output:

**MIP-DNA**

Screen Shot 2022-05-31 at 17 38 15

**BALSAMIC**

Screen Shot 2022-05-31 at 17 39 09

### Updated query output:

**MIP-DNA**
Screen Shot 2022-05-31 at 17 42 57

**BALSAMIC**

Screen Shot 2022-05-31 at 17 41 41

File: cg/store/api/status.py [hunk 1]
```diff
@@ -634,8 +634,8 @@ def analyses_to_delivery_report(self, pipeline: Pipeline = None) -> Query:
             .filter(VALID_DATA_IN_PRODUCTION < models.Analysis.started_at)
             .join(models.Family, models.Family.links, models.FamilySample.sample)
             .filter(
-                or_(
-                    models.Family.data_analysis.is_(None),
+                and_(
+                    models.Analysis.pipeline == str(pipeline),
                     models.Family.data_analysis == str(pipeline),
                 )
```
Comment (patrikgrenfeldt) [inline cg/store/api/status.py:640]:
```suggestion
                models.Analysis.pipeline == str(pipeline),
```
This should also give a valid result?

**Review by patrikgrenfeldt — APPROVED** [review]
🤔

**Comment by ivadym** [comment]
Screen Shot 2022-06-01 at 11 42 51
Screen Shot 2022-06-01 at 11 45 44

**Pull request merged**